crates/cheatcodes/src/json.rs

```diff
@@ -431,6 +446,10 @@ pub(super) fn value_to_token(value: &Value) -> Result<DynSolValue> {
                     32 => DynSolValue::FixedBytes(B256::from_slice(&bytes), 32),
                     _ => DynSolValue::Bytes(bytes),
                 })
+            } else if string == "true" {
+                Ok(DynSolValue::Bool(true))
+            } else if string == "false" {
+                Ok(DynSolValue::Bool(false))
```

Comment:
Why was this added?

Reply:
Good catch, I added it to implicitly handle bool conversion in the same way we handle the `null` case when I was improving the test suite. Looking at it again it is better to not implement it and instead defer to the original implementation where users can use type coercion to cast this to a bool.

I've made some improvements to the boolean tests of JSON and TOML to make sure the existing implementation is correct.